users/pdl/pdl.h

```diff
@@ -0,0 +1,115 @@
+#pragma once
+
+#include "quantum.h"
+
+enum userspace_layer_codes {
+    QWERTY = SAFE_RANGE,
+    PROXIM,
+    PUNCTN,
+    CODING,
+    KU_LBRC,
+    KU_NUHS,
+    KU_GRV,
+    KU_RBRC,
+    NEW_SAFE_RANGE
+};
+
+#define _QWERTY 2
+#define _PROXIM 0
+#define _NUMBRS 4
+#define _NAVIGN 5
+#define _PUNCTN 6
+#define _CODING 7
+#define _FUNCTN 8
```

Comment:
Just a heads up, we recommend enums for layer number, since skipping numbers like this will bloat the firmware size.